### ISSUE TITLE
Don't show < when value is exact

### ISSUE THREAD

**Issue by flyswatter** [MetaMask/metamask-plugin#472]
Our number formatter acts a little strange with some values.

With this example, the value is exactly `0.001` eth.

<img width="98" alt="screen shot 2016-07-20 at 3 19 34 pm" src="https://cloud.githubusercontent.com/assets/542863/17005148/7f736d7c-4e8d-11e6-9654-8ff18ece5c27.png">

In this case, there should be no `<` symbol, since it is exactly the correct price, and arguably it shouldn't use scientific notation either, since it's fewer digits than the scientific notation syntax itself.

Also, the hover value should not display more digits than the precision requires.

All of these criteria should be added to the number formatter test suite.

**Comment by Gustav-Simonsson** [comment]
Any thoughts on localisation?
One idea could be to localize the formatting based on chrome's locale according to
http://www.thefinancials.com/Default.aspx?SubSectionID=curformat
https://en.wikipedia.org/wiki/ISO_4217

For non-technical users seeing a formatting in style of mainstream financial UIs rather than scientific notation can increase confidence and trust in the extension.

**Comment by flyswatter** [comment]
Those are great ideas for formatting known currencies, but they don't really solve the problems of formatting new currencies like Ether, do they?

Do you have any suggestions for how you would want a miniscule amount of ether represented in a list of transactions? Or an amount of ether with a level of precision far beyond what any user practically cares about?

**Comment by Gustav-Simonsson** [comment]
I think they do, for a user interface like this I'd consider formatting based on number of Ether, not Wei, as per the user's locale. So e.g. round up to 0.01 Ether if the user has less than that.

**Comment by flyswatter** [comment]
I'm a little confused, Ether & Wei don't seem like locale-based currencies to me. I can't imagine any region preferring Wei.

I'm also kindof on the fence about rounding up. Is that really more honest? I guess it's probably safe just because it's a small enough quantity that hopefully people aren't relying on it, I guess I'm just very hesitant about making that call for all users, I don't know how precise some users might require.

I guess as long as we have a way to copy the full-precision value, it doesn't matter as much. I do agree that scientific notation and less-than symbols are clunky, and definitely not the kind of thing people expect from financial instruments.

So to summarize, the proposal is:

Round up to 2 or maybe 3 decimals, leave it at that.

That works fine for small values, it doesn't *quite* solve our extreme edge cases where people are using `ethereumjs-testrpc` and have a million ether (we start to run out of space), but maybe we can just scale the font down at that point.

I'm also leaning towards a hover tooltip that has a higher precision value and a "copy" note, so if you click it switches to "copied" to indicate you've copied the full precision balance in eth.
